functions/Get-DbaSqlBuildReference.ps1

```diff
@@ -0,0 +1,210 @@
+Function Get-DbaSqlBuildReference
+{
+<#
+.SYNOPSIS
+Returns SQL Server Build infos on a SQL instance
+
+.DESCRIPTION
+Returns info about the specific build of a SQL instance, including the SP, the CU and the reference KB, wherever possible.
+It also includes End Of Support dates as specified on Microsoft Lifecycle Policy
+
+.PARAMETER SqlInstance
+The SQL Server that you're connecting to.
+
+.PARAMETER Credential
+Credential object used to connect to the SQL Server as a different user.
+
+.PARAMETER Build
+Instead of connecting to a real instance, pass a string identifying the build to get the info back.
+
+.PARAMETER Silent
+Replaces user friendly yellow warnings with bloody red exceptions of doom!
```

Comment:
straight copy-pasted, sorry. will try to come up with something better but, really, this would need to be documented and then copied over any "silent" parameter spec in cmdlets using the new logging infrastructure